servo/servo.py

```diff
@@ -132,7 +132,6 @@ async def adjust(
     async def promote(self) -> None:
         ...
 
-
```

Comment:
Sorry for another nitpick but other classes in this file have two whitespace lines between them. Please remove this

Reply:
We need `black`, this is silly.